numpy/linalg/linalg.py

```diff
@@ -1226,8 +1226,9 @@ def svd(a, full_matrices=1, compute_uv=1):
     """
     Singular Value Decomposition.
 
-    Factors the matrix `a` as ``u * np.diag(s) * v``, where `u` and `v`
-    are unitary and `s` is a 1-d array of `a`'s singular values.
+    Factors the matrix `a` as ``u * S * v``, where `u` and `v` are unitary and 
+    `S` is a matrix with shape ``(a.shape[0], a.shape[1])`` with 
```

Comment:
I think this should more emphatically distinguish the array and matrix cases, with the array version getting more prominence.